dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/DdAndroidGradlePlugin.kt

```diff
@@ -80,10 +80,10 @@ class DdAndroidGradlePlugin @Inject constructor(
         variant: ApplicationVariant,
         apiKey: ApiKey
     ) {
-        if (isObfuscationEnabled(variant, datadogExtension)) {
-            val buildIdGenerationTask =
-                configureBuildIdGenerationTask(target, androidExtension, variant)
+        val buildIdGenerationTask =
```

Comment:
Yeup exactly. Before it was only needed if obfuscation was enabled because that's the only way you'd get a mapping file. Now, you can get NDK symbol files even if obfuscation isn't enabled.